runtime/src/main/java/org/corfudb/runtime/view/ClusterStatusReport.java

```diff
@@ -17,15 +17,15 @@
     /**
      * Connectivity to the node.
      */
-    enum NodeStatus {
+    public enum NodeStatus {
         UP,
         DOWN
     }
 
     /**
      * Health of the cluster.
      */
-    enum ClusterStatus {
+    public enum ClusterStatus {
```

Comment:
This also exposes the health value to the clients. Are we ok with committing to stick with these health values?
If not, we have to at least highlight this in their java docs.